traefik/crds/kustomization.yaml

```diff
@@ -1,24 +0,0 @@
----
-kind: Kustomization
-apiVersion: kustomize.config.k8s.io/v1beta1
-resources:
-  # curl -o gateway-standard-install-v1.1.0.yaml -L https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
-  - gateway-standard-install-v1.1.0.yaml
-  - hub.traefik.io_accesscontrolpolicies.yaml
-  - hub.traefik.io_apiaccesses.yaml
-  - hub.traefik.io_apiportals.yaml
-  - hub.traefik.io_apiratelimits.yaml
-  - hub.traefik.io_apis.yaml
-  - hub.traefik.io_apiversions.yaml
-  - hub.traefik.io_apiplans.yaml
-  - hub.traefik.io_apibundles.yaml
-  - traefik.io_ingressroutes.yaml
-  - traefik.io_ingressroutetcps.yaml
-  - traefik.io_ingressrouteudps.yaml
-  - traefik.io_middlewares.yaml
-  - traefik.io_middlewaretcps.yaml
-  - traefik.io_serverstransports.yaml
-  - traefik.io_serverstransporttcps.yaml
-  - traefik.io_tlsoptions.yaml
-  - traefik.io_tlsstores.yaml
```

Comment:
This file should not be removed. 
It's the current working & documented way to upgrade CRDs.

Reply:
I documented it.